bundles/io/org.eclipse.smarthome.io.transport.mdns/src/main/java/org/eclipse/smarthome/io/transport/mdns/internal/MDNSClientImpl.java

```diff
@@ -226,4 +261,29 @@ public void onChanged(List<CidrAddress> added, List<CidrAddress> removed) {
             createJmDNSByAddress(address);
         }
     }
+
+    private void getConfigParameters(Map<String, Object> parameters) {
+        useOnlyOneAddress = getConfigParameter(parameters, CONFIG_USER_ONLY_ONE_ADDRESS, false);
```

Comment:
Should default to true in my opinion

Reply:
I didn't want to change the default behaviour.